packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -768,16 +788,22 @@ function WPLineItem( {
 } ): JSX.Element {
 	const id = product.uuid;
 	const translate = useTranslate();
-	const hasDomainsInCart = responseCart.products.some(
-		( product ) => product.is_domain_registration || product.product_slug === 'domain_transfer'
+	const hasBundledDomainsInCart = responseCart.products.some(
+		( product ) =>
+			( product.is_domain_registration || product.product_slug === 'domain_transfer' ) &&
+			product.is_bundled
+	);
+	const hasMarketplaceProductsInCart = responseCart.products.some(
```

Comment:
Good idea, resolving the product type from the source (wpcom) rather than from Calypso. It let's us package the code without Calypso dependencies.